src/polyfills/event.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Sets the Event polyfill if it does not exist.
+ * @param {!Window} win
+ */
+export function install(win) {
+  // win.Event is a function on Edge, Chrome, FF, Safari but
+  // is an object on IE 11.
+  if (typeof win.Event === 'function') {
+    return;
+  }
+
+  function Event(name, params) {
```

Comment:
The current Travis error is a Closure compiler type error that seems to dislike the fact that this isn't a new-only constructor function (i.e. ES6 class). I'd try fixing it by changing this to be an ES6 class. If that doesn't work, type-cast. 😄 

You can re-run the type checker locally by running `gulp check-types`.